src/scripts/delete.js

```diff
@@ -0,0 +1,28 @@
+import { task } from './addnew.js';
+import storageContainer from './htmldisplay.js';
+
+// export const taskDelete = () => {
+//     const clearButton = document.getElementById('clear');
+//     clearButton.addEventListener('click', () => {
+//         const result = task.filter((x) => x.complete === false);
+//         task.splice(0);
+//         task.push(...result);
+//         localStorage.setItem('task', JSON.stringify(task));
+//         storageContainer();
+//     });
+// };
```

Comment:
- [ ] Please Do not commit old pieces of code as inline comments. They will make your project look unprofessional.

Reply:
they are not old pieces of code, they are actually parts of the next requirements that I've commented